### PR TITLE
Fix newly added cancel links

### PR DESCRIPTION
**Pull request by dnkrj**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

The cancel links added in https://github.com/alphagov/whitehall/pull/6941 do not work.
This PR uses the link_to function to generate the anchor tags.

### CODE REVIEW

**Review by ollietreend — APPROVED** [review]

File: app/views/admin/edition_tags/edit.html.erb [hunk 1]
```diff
@@ -72,6 +72,7 @@
           }
         } %>
 
+        <%= link_to("Cancel", admin_edition_path(@edition), class: "govuk-link") %>
         <a class="govuk-link" href="<%= admin_edition_path(@edition) %>">Cancel</a>
```
Comment (davidgisbey) [inline app/views/admin/edition_tags/edit.html.erb:76]: I think this needs to come out then right?

File: app/views/admin/attachments/confirm_destroy.html.erb [hunk 2]
```diff
@@ -2,6 +2,11 @@
 <% content_for :page_title, "Delete attachment" %>
 <% content_for :title, "Delete attachment" %>
 <% content_for :title_margin_bottom, 6 %>
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: attachable_attachments_path(attachable)
+  } %>
+<% end %>
```
Comment (davidgisbey) [inline app/views/admin/attachments/confirm_destroy.html.erb:9]: Do we def want a backlink and cancel link on the same page?

Comment (dnkrj) [reply app/views/admin/attachments/confirm_destroy.html.erb:9]: Oh, good catch, sorry, have removed it again

File: app/views/admin/edition_tags/edit.html.erb (hunk 1 above)
Comment (dnkrj) [reply app/views/admin/edition_tags/edit.html.erb]: Thanks!

**Review by davidgisbey — APPROVED** [review]

**Pull request merged**